### PR TITLE
Script for adding Additional Service

### PR DESCRIPTION
**Pull request by michaelpace-bjss**

### CODE REVIEW

File: Import/AdditionalServiceImport.sql [hunk 1]
```diff
@@ -0,0 +1,142 @@
+
+IF type_id('[dbo].[AdditionalServiceImportCapability]') IS NULL
+CREATE TYPE [dbo].[AdditionalServiceImportCapability] as TABLE (CapabilityRef varchar(10)) 
+GO
+
+CREATE OR ALTER PROCEDURE AddtionServiceImport
```
Comment (stuart-bass-cgi) [inline Import/AdditionalServiceImport.sql]: typo - AdditionalServiceImport

File: Import/Tests/AdditionalServiceExample.sql [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+print @SupplierId 
+print @ParentSolutionId 
+print @AdditionServiceId 
+print @ServiceName 
+print @ServiceSummary 
+print @ServiceDescription
+
+select * from @Capabilities
+
+EXEC AddtionServiceImport
```
Comment (stuart-bass-cgi) [inline Import/Tests/AdditionalServiceExample.sql]: matching typo

File: Import/Tests/AdditionalServiceExample.sql [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+DECLARE @ServiceName varchar(255)
+DECLARE @ServiceSummary varchar(300)
+DECLARE @ServiceDescription varchar(3000)
+DECLARE @Capabilities AdditionalServiceImportCapability
+
+/***
+SET DATA HERE
+***/
+
+SET @SupplierId = '10000'
```
Comment (stuart-bass-cgi) [inline Import/Tests/AdditionalServiceExample.sql]: In our test data, notEmisWeb GP  is Solution Id 99999-89, SupplierId 99999. Can these please be adjusted to match the test data?

File: Import/AdditionalServiceImport.sql [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+		THROW 51000, 'Parent Solution record does not exist.', 1;  
+
+	SELECT @PublishedStatusId=PublishedStatusId, @AuthorityStatusId = AuthorityStatusId, @SupplierStatusId = SupplierStatusId
+		FROM [dbo].[Solution] WHERE [Id] =  @ParentSolutionId;
+
+	/***
+	Add the Addition Service if not already present
+	***/
+	IF NOT EXISTS (SELECT 1 FROM [dbo].[Solution] WHERE [Id] =  @AdditionServiceId)	
+	INSERT INTO [dbo].[Solution]
```
Comment (stuart-bass-cgi) [inline Import/AdditionalServiceImport.sql:54]: Is a Version Required?

**Review by stuart-bass-cgi — CHANGES_REQUESTED** [review]

File: Import/AdditionalServiceImport.sql (hunk 4 above)
Comment (michaelpace-bjss) [reply Import/AdditionalServiceImport.sql:54]: Version not required. The version is information field defined by the supplier for the parent solution. Two versions of a solution will have different solution ids.

File: Import/AdditionalServiceImport.sql (hunk 1 above)
Comment (michaelpace-bjss) [reply Import/AdditionalServiceImport.sql]: Fixed

File: Import/Tests/AdditionalServiceExample.sql (hunk 2 above)
Comment (michaelpace-bjss) [reply Import/Tests/AdditionalServiceExample.sql]: Fixed

File: Import/Tests/AdditionalServiceExample.sql (hunk 3 above)
Comment (michaelpace-bjss) [reply Import/Tests/AdditionalServiceExample.sql]: Fixed

**Review by stuart-bass-cgi — APPROVED** [review]

**Pull request merged**